tools/HiveCompare/HiveCompare/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿using HiveCompare.Models;
+using Microsoft.Extensions.CommandLineUtils;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+
+internal class Program
+{
+    public static JsonSerializerOptions SERIALIZER_OPTIONS = new JsonSerializerOptions
+    {
+        PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+        WriteIndented = true,
+    };
+    private static void Main(string[] args)
+    {
+        var cli = CreateCommandLineInterface();
+        try
+        {
+            cli.Execute(args);
+        }
+        catch (CommandParsingException)
+        {
+            cli.ShowHelp();
+        }
+    }
+
+    static CommandLineApplication CreateCommandLineInterface()
```

Comment:
Can you split this method in more digestable parts?

Reply:
Do you think it still needs splitting after the cleanup?